synapse/config/experimental.py

```diff
@@ -84,7 +84,7 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         # experimental support for faster joins over federation
         # (MSC2775, MSC3706, MSC3895)
         # requires a target server that can provide a partial join response (MSC3706)
-        self.faster_joins_enabled: bool = experimental.get("faster_joins", False)
+        self.faster_joins_enabled: bool = experimental.get("faster_joins", True)
```

Comment:
As a sanity check, this will:

1. Send both the stable and unstable query args in the request to join a room. Old versions or versions with this flag disabled will ignore these args and do a full join.
2. Enabling handling of the unstable response keys. These *should* only get sent if the args are provided.

Is that it?

Reply:
1 is exactly correct.

2. Is correct too. The _should_ is enforced by https://github.com/matrix-org/synapse/blob/d329a566df6ff2b635a375bf1b2c8ed3b2c9815d/synapse/federation/federation_client.py#L1155-L1163 in the bit of logic which post-processes the `SendJoinResponse` into a `SendJoinResult`.

Reply:
That sounds great! 👍